main.go

```diff
@@ -153,6 +184,74 @@ func (zarinpal *Zarinpal) PaymentVerification(amount int, authority string) (ver
 	return
 }
 
+// UnverifiedTransactions gets unverified transactions.
+//
+// If error is not nil, you can check statusCode for
+// specific error handling based on Zarinpal error codes.
+// If statusCode is not 100, it means Zarinpal raised an error
+// on their end and you can check the error code and its reason
+// based on their documentation placed in
+// https://github.com/ZarinPal-Lab/Documentation-PaymentGateway/archive/master.zip
+func (zarinpal *Zarinpal) UnverifiedTransactions() (statusCode int, authorities []UnverifiedAuthority, err error) {
```

Comment:
@Mahmoud-Sagharjoughi Its better to return authorities as a first return argument.

Reply:
@sinabakh It Fixed.